.github/workflows/release-layer-collector.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           path: ${{ github.workspace }}/collector/build/*.zip
 
       - name: Save Collector Version
+        if: ${{ matrix.architecture == 'amd64' }}
```

Comment:
Do the builds run in order, or do we need to have a dependency to ensure the `amd64` build runs first and not in parallel?

Reply:
Since publish `needs: [build-layer]` i think it should be fine